src/redis_stack/probabilistic_data_structures.md

```diff
@@ -1,27 +1,39 @@
-In very broad terms probabilistic data structures (PDS) allow us to get to a "close enough" result in a much shorter time and by using significantly less memory.
+This tutorial will demonstrate Redis Stack's probabilistic data structure capabilities using the bike shop use case.
 
-Redis Stack supports 4 of the most famous PDS:
-- Bloom filters
-- Cuckoo filters
-- Count-Min Sketch
+Redis Stack supports the following probabilistic data structures:
+
+- Bloom filter
+- Cuckoo filter
+- Count-min sketch
```

Comment:
we can use something like "In addition to basic data structures, Redis supports the following advanced probabilistic data structures" to use Redis instead of Redis Stack

Reply:
HyperLogLog is also a probabilistic data structure. Though it is a "basic data structure" we should add it to any list of probabilistic data structures.

Reply:
Done. See next push.